cluster_tools/Changelog.md

```diff
@@ -10,10 +10,13 @@ For upgrade instructions, please check the respective *Breaking Changes* section
 [Commits](https://github.com/scalableminds/webknossos-libs/compare/v0.13.6...HEAD)
 
 ### Breaking Changes
+- Dropped support for Python 3.7. [#943](https://github.com/scalableminds/webknossos-libs/pull/943)
```

Comment:
how about recommending `exc.as_completed` instead of `concurrent.futures.as_completed` since the latter won't work for dask futures, if I see this correctly?